.github/workflows/build.yml

```diff
@@ -56,6 +56,13 @@ jobs:
           sudo apt-get install -y autoconf automake pkg-config g++-mingw-w64 gcc-multilib python3-pip ninja-build nasm wine wine32 wine64 ccache
           sudo python3 -m pip install meson
 
+      - name: Install llvm-mingw
+        run: |
+          export var=20240320
```

Comment:
`ver`? also put this in `env:` block for the job.

Reply:
Why not use the [release-downloader](https://github.com/robinraju/release-downloader) action, and always download the latest version?